currency_rate_update/models/res_currency_rate_provider.py

```diff
@@ -278,9 +293,22 @@ def _scheduled_update(self):
                     if provider.last_successful_run
                     else (provider.next_run - provider._get_next_run_period())
                 )
+                newest_only = True
                 date_to = provider.next_run
-                provider._update(date_from, date_to, newest_only=True)
-
+                current_utc_hour = datetime.now().hour
+                _logger.debug(
+                    "Provider %s date_to=%s today=%s, current hour %s UTC",
+                    provider.name,
+                    date_to,
+                    today,
+                    current_utc_hour,
+                )
```

Comment:
I don't see these lines in other versions, but I am guessing that it is an intended change for logging purpose.

Reply:
This code currently exists in v14 (https://github.com/OCA/currency/blob/14.0/currency_rate_update/models/res_currency_rate_provider.py#L300-L307)
It was added at https://github.com/OCA/currency/pull/135 to 14.0 but was not ported to v15.